proposals/N3GOV-X.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": 20,
+  "spec": 1,
+  "votes": [
+    null
+  ],
+  "content": {
+    "kind": "expense",
+    "transfers": [
+      {
+        "chainId": 1,
+        "from": "0x336252602b3a8a0be336ed942228305173e8082b",
+        "recipient": "0x7fFf737ae9FdDD6E28Fe8B8F0F4dE69Bb8e7a885",
+        "token": "0x333A4823466879eeF910A04D473505da62142069",
+        "amount": 75
```

Comment:
@anastasiyabelyaeva Not 50, as discussed in the forum thread?  https://forum.nation3.org/t/nation3-flag-campaign-to-activate-the-community-and-allocate-50-of-the-remaining-genesis-passports/755/14?u=aahna

Reply:
Sorry, misread the number.  50 citizens, 75 tokens.